documentation/tutorials/get-started.md

```diff
@@ -110,23 +110,26 @@ defp deps do
 end
 ```
 
-### Creating our first resources
+### Building your first Ash API
 
-The basic building blocks of an Ash application are resources. They are tied together by an API module (not to be confused with a web API), which will allow you to interact with those resources.
+The basic building blocks of an Ash application are Ash resources. They are tied together by an API module (not to be confused with a web API), which will allow you to interact with those resources.
+
+It might be helpful to think of an Ash API as a Bounded Context (in the Domain Driven Design sense), or as a Service (in the microservice sense).
```

Comment:
I like the bounded context callout, but equating an Ash Api to a microservice might give people the wrong idea.